modules/gitops-accessing-gitops-monitoring-dashboards.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-with-gitops-dashboards.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-gitops-monitoring-dashboards_{context}"]
+= Accessing {gitops-shortname} monitoring dashboards
+
+The monitoring dashboards are deployed automatically by the Operator. You can access {gitops-shortname} monitoring dashboards from the *Administrator* perspective of the {OCP} web console. 
+
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default namespace, `openshift-gitops-operator`.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Observe* -> *Dashboards*.
+
+. From the *Dashboard* drop-down list, select the desired {gitops-shortname} dashboard: *{gitops-shortname} (Overview)*, *{gitops-shortname} / Components*, or *{gitops-shortname} /  gRPC Services*.
+
+. Optional: Choose a specific namespace, cluster, and interval from the *Namespace*, *Cluster*, and *Interval* drop-down lists.
```

Comment:
@Srivaralakshmi Yes I tested this step, but in my cluster version I did not see all these options, I explain it here in this comment: https://docs.google.com/document/d/1QAHaub04AYH3-LiXqAveXmeM2V-ZyFD2Q6t6ECBWr-A/edit?disco=AAAA6eULc4Q
However, the SME in the same comment confirmed that the options are as described in this step, there are added screenshots in the draft where the comment is.

Reply:
Hello @reginapizza I had a discussion with Sri about the issue of not seeing the options to choose namespace and cluster while testing the GitOps dashboards. Here is my screenshot: 
![image](https://github.com/openshift/openshift-docs/assets/98941675/3f7bf8d3-02bf-4d2b-86e5-04b887b3ed1a)
As you can see, I can only choose an interval. But I saw that you had those options in the sceenshot you provided.  I have tested it with ocp 4.15 image and with rosa as well and I could not see the options in either of those. We need to know why is that a case before we can merge the issue to docs. Can you help me find out what the problem might be? Thank you!

Reply:
@ekristova ok I think I know the issue... when you have multiple argocd instances it seems that it will pick up on that and then give you the namespace dropdown to either choose "all" or the "openshift-gitops" namespace. Try adding another argocd instance in another namespace and see if you can view that dropdown again.

Reply:
@reginapizza I have tried to add multiple argocd instances in other namespaces as you suggested and it still does not work :(  can there be another issue? 
cc: @varshab1210 could you possibly also help me figure out what might be wrong? Thank you!

Reply:
Hello @Srivaralakshmi I got on a call with @reginapizza to solve the issue, basically what we were missing was to create new argocd instance in a different namespace and deploy an application in that instance. Then the dashboards pick up the data and show the dropdowns for namespaces and clusters.
Here is the screenshot when I added applications to "example" and "openshift-gitops" namespaces:
![image](https://github.com/openshift/openshift-docs/assets/98941675/128cbf50-f627-43a0-8230-1a473e0f5b34).
So, I think the easy solution for our docs would be to add this to the prerequisites:
* You have installed an Argo CD application in your defined namespace, for example, `openshift-gitops`.
WDYT?

Reply:
@ekristova yes! Add something along these lines: You have created a new Argo CD instance in your defined namespace, for example, `openshift-gitops`, and deployed an application in that instance.

Reply:
@Srivaralakshmi  I think it is also enough to deploy an application on the instance that is created automatically with installation. Then the prereq would be only the creation of an application. I test that theory and make the prerequisite accordingly.

Reply:
@ekristova Yes, I remember stumbling on that observation last time when I tested and shared it with you. I just tested again to confirm it. This prerequisite is good enough: You have installed an Argo CD application in your defined namespace, for example, openshift-gitops. Thanks!